app/scenes/Login/Service.js

```diff
@@ -0,0 +1,143 @@
+// @flow
```

Comment:
🍖  Signin service, this is built with an eye on the future where signin services will be more "pluggable"